src/Home/Home.tsx

```diff
@@ -1,5 +1,80 @@
 import * as React from 'react';
+import glamorous from 'glamorous';
+import './Home.css';
+const { Div } = glamorous;
+import PageCard from './PageCard';
+import CircleButton from './CircleButton';
 
-const Home = () => <h1>Home Page</h1>;
+class Home extends React.Component {
+  render() {
+    return (
+      <Div margin="5px auto" padding="20px" width="100%">
+        <Div id="nav" display="flex">
```

Comment:
You can use the semantic [nav](https://htmlreference.io/element/nav/) element here instead of a div!